src/Protocol/Diaspora/Repository/DiasporaContact.php

```diff
@@ -234,6 +234,18 @@ public function getByUrl(UriInterface $uri, ?bool $update = self::UPDATE_IF_MISS
 	 */
 	public function updateFromProbeArray(array $data): Entity\DiasporaContact
 	{
+		if (empty($data['url'])) {
+			throw new \InvalidArgumentException('Missing url key in Diaspora probe data array');
```

Comment:
Please include some logging with the `$data` and the provided url, so that we can dig into it.

Reply:
This is done in the `catch` blocks of the few places it is called from.